packaging/linux/run_keybase

```diff
@@ -64,8 +68,15 @@ kill_all() {
   if killall Keybase &> /dev/null ; then
     echo Shutting down Keybase GUI...
   fi
+  # In case the package upgrade wasn't able to unmount and remove /keybase
   if fusermount -uz /keybase &> /dev/null ; then
-    echo Unmounting /keybase...
+    # Remove any existing legacy mount.
+    echo Unmounting /keybase and converting to symlink...
+    rmdir /keybase
+    ln -s /opt/keybase/mount-readme /keybase
```

Comment:
Yeah good point. This wouldn't work, and shouldn't be necessary since `post_install.sh` should have already taken care of it.  It was meant as a catch-all, but it's useless.  I'll print something out instead I guess.